docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  runner:
+    build:
+      context: .
+      args:
+        RUBY_VERSION: ${RUBY_VERSION:-2.3.1}
+        BUNDLER_VERSION: ${BUNDLER_VERSION:-}
+    volumes:
+      - type: bind
+        source: .
+        target: /app
```

Comment:
mentioned elsewher e- nitpick but you might want to use this - https://docs.docker.com/reference/compose-file/services/#working_dir to avoid confusion if the dockerfile itself ever changes.

super nitpick, approving PR anyway should you choose to disregard this. happy to reapprove if you do make this change!